code/game/objects/items/weapons/tools/weldingtools.dm

```diff
@@ -31,6 +31,7 @@
 		..()
 		damtype = BURN
 		force = force_ignited
+		ignites_with_touch = 1
```

Comment:
use TRUE/FALSE instead of 1/0

Reply:
The variable can be potentially used to represent temperature for ignition purposes. I use 1 and 0 everywhere for clarity since there is no boolean type in byond and it is misleading.